lms/djangoapps/badges/tests/test_models.py

```diff
@@ -157,7 +159,7 @@ def test_get_badge_class_create(self):
         self.assertEqual(badge_class.description, 'This is a test')
         self.assertEqual(badge_class.criteria, 'https://example.com/test_criteria')
         self.assertEqual(badge_class.display_name, 'Super Badge')
-        self.assertEqual(badge_class.image.name.rsplit('/', 1)[-1], 'good.png')
+        self.assertTrue('good' in badge_class.image.name.rsplit('/', 1)[-1])
```

Comment:
What kind of value were you getting that didn't work with the original assertion?

Reply:
The returned values changed to include the hash when the file was found to be in the media root, so they are now things like `good_DLu2gWT.png`. This should be how our files are usually stored, so I think this just brings it in line with how things actually work in prod.